src/exporter/DaemonMetricCollector.cc

```diff
@@ -298,20 +311,21 @@ labels_t DaemonMetricCollector::get_extra_labels(std::string daemon_name) {
   if (daemon_name.rfind(ceph_client_prefix, 0) == 0) {
     daemon_name = daemon_name.substr(ceph_client_prefix.size());
   }
-  // In vstart cluster socket files for rgw are stored as radosgw.<instance_id>.asok
+  // In vstart cluster socket files for rgw are stored as
+  // radosgw.<instance_id>.asok
   if (daemon_name.find("radosgw") != std::string::npos) {
     std::size_t pos = daemon_name.find_last_of('.');
-    std::string tmp = daemon_name.substr(pos+1);
+    std::string tmp = daemon_name.substr(pos + 1);
     labels["instance_id"] = quote(tmp);
-  }
-  else if (daemon_name.find("rgw") != std::string::npos) {
-    // fetch intance_id for e.g. "hrgsea" from daemon_name=rgw.foo.ceph-node-00.hrgsea.2.94739968030880
+  } else if (daemon_name.find("rgw") != std::string::npos) {
+    // fetch intance_id for e.g. "hrgsea" from
+    // daemon_name=rgw.foo.ceph-node-00.hrgsea.2.94739968030880
     std::vector<std::string> elems;
     std::stringstream ss;
     ss.str(daemon_name);
     std::string item;
     while (std::getline(ss, item, '.')) {
-        elems.push_back(item);
+      elems.push_back(item);
```

Comment:
if you are not changing this part of code, i'd suggest avoid this kind of formatting only change.

Reply:
done!